### ISSUE TITLE
Note on the docs

### ISSUE THREAD

**Issue by grobmeier** [franzliedke/studio#66]
Franz, thank you for Studio, it helped me greatly today.

here is a note which you may want to use for the docs (if valid): I was only able to get studio to work when I changed the version to dev-develop (rather than (^2.0.0 in example). Only then the symlinks got created.

I don't know if this is by intention so I just I would drop an issue here.

Thanks again! Christian

**Comment by franzliedke** [comment]
Hi Christian, thanks for getting in touch!

Is this the same problem that is mentioned [here](https://github.com/franzliedke/studio/issues/56#issuecomment-254808455)?

**Comment by grobmeier** [comment]
Hi Franz, not sure, but I think not. Here is what I did:

1. Created a project with "normal dependencies" from Satis. Using my/world-domination ^2.0.0
2. composer install
3. Learned about Studio :-) Did: composer require --dev franzl/studio
4. studio load path/to/world-domination
5. composer update my/world-domination

At this point I expected my "old dependency" to be gone, but it was not. No symlinks created.

6. Changed version from ^2.0.0 to dev-develop
7. composer update my/world-domination

At this point it worked as expected.

I believe nothing happened because there was no change recognized (same version etc). If this is actually the case, it would be worth pointing it out in the docs

Thanks for taking the time!
